vm/chef/cookbooks/mariadb/files/mariadb-utils

```diff
@@ -0,0 +1,323 @@
+#!/bin/bash
+
+source /opt/c2d/c2d-utils || exit 1
+
+declare -r replication_user="replication"
+declare -r backup_folder="/tmp/transfer"
+declare -r backup_filename="${backup_folder}/all-databases.sql"
+
+#######################################
+# Prints a formatted message to standard output.
+# Arguments:
+#  Message string.
+#######################################
+function emit_msg() {
```

Comment:
maybe 'log' is more straightforward name

Reply:
replace with echo

Reply:
Done.